python/semantic_kernel/connectors/ai/ollama/services/ollama_text_completion.py

```diff
@@ -31,6 +31,7 @@ class OllamaTextCompletion(TextCompletionClientBase, AIServiceClientBase):
         url {Optional[Union[str, HttpUrl]]} -- URL of the Ollama server, defaults to http://localhost:11434/api/generate
     """
 
+    ai_model_id: str = "llama2"
```

Comment:
do we need this and I would like to keep the annotations on the ai_model_id field from AIServiceClientBase: `ai_model_id: Annotated[str, StringConstraints(strip_whitespace=True, min_length=1)]`